metaci/repository/utils.py

```diff
@@ -1,48 +1,63 @@
+from metaci.build.exceptions import BuildError
+
+
 def create_status(build):
     if not build.plan.context:
         # skip setting Github status if the context field is empty
         return
 
-    repo = build.repo.github_api
+    state = None
+    target_url = None
+    description = None
+    repo = build.repo.get_github_api()
 
     if build.get_status() == "queued":
         state = "pending"
         description = "The build is queued"
-    if build.get_status() == "waiting":
+
+    elif build.get_status() == "waiting":
```

Comment:
This is a tiny thing and it was already there, but let's do `status = build.get_status()` once and then use it so we aren't repeatedly calling get_status()